index.js

```diff
@@ -1,11 +1,28 @@
 var N = require('libnested')
 
 var isModule = require('./is')
+var apply = require('./apply')
+
+module.exports = function combine () {
+  var modules = flattenNested(arguments)
```

Comment:
i think we should call `Array.prototype.slice.call(arguments)` (coerce `arguments` to an `Array`) here and not in `flattenNested`